packages/loopring_v3/contracts/impl/ProtocolFeeManager.sol

```diff
@@ -183,48 +196,53 @@ contract ProtocolFeeManager is IProtocolFeeManager, Claimable
         IAuction(auction).settle();
     }
 
-    // TODO(dongw): this method is not implementated.
     function auctionOffTokens(
-        address tokenS,
+        address token,
+        uint    amount,
         bool    sellForEther,
-        uint64  ,//P,
-        uint64  ,//S,
-        uint8   ,//M,
-        uint    //T
+        uint    minAskAmount,
+        uint    minBidAmount,
+        uint64  P,
+        uint64  S,
+        uint8   M,
+        uint    T
         )
         external
         onlyOwner
         returns (
-            address payable //auctionAddr
+            address payable auctionAddr
         )
     {
         require(oedaxAddress != address(0), "ZERO_ADDRESS");
-
+        require(amount > 0, "ZERO_AMOUNT");
         address tokenB = sellForEther ? address(0) : lrcAddress;
-        require(tokenS != tokenB, "SAME_TOKEN");
-
-        // IOedax oedax = IOedax(oedaxAddress);
-        // uint ethStake = oedax.creatorEtherStake();
-
-        // auctionAddr = oedax.createAuction.value(ethStake)(
-        //     tokenS,
-        //     tokenB,
-        //     0,
-        //     0,
-        //     P,
-        //     S,
-        //     M,
-        //     T,
-        //     T * 2
-        // );
-
-        // IAuction auction = IAuction(auctionAddr);
-
-        // auction.ask(124);
-
-        // emit AuctionStarted(
-        //     tokenS,
-        //     auctionAddr
-        // );
+
+        IOedax oedax = IOedax(oedaxAddress);
+        uint ethStake = oedax.creatorEtherStake();
+
+        auctionAddr = oedax.createAuction.value(ethStake)(
+            token,  // askToken
+            tokenB, // bidToken
+            minAskAmount,
+            minBidAmount,
+            P,
+            S,
+            M,
+            T,
+            T * 2
+        );
+
+        if (token == address(0)) {
+            auctionAddr.transfer(amount);
```

Comment:
`address.transfer` has a gas limit of 2300 gas which is probably too low to do the additional logic in the default function. Transferring the ETH like this doesn't have that gas limit:
```
(success, ) = auctionAddr.call.value(amount)("");
```

Reply:
great.